packages/firestore/src/api/database.ts

```diff
@@ -296,6 +321,32 @@ export class Firestore implements firestore.FirebaseFirestore, FirebaseService {
       'FirestoreSettings.host cannot be falsey'
     );
 
+    if (!this._config.settings.timestampsInSnapshotsEnabled) {
+      log.error(`
+The default behavior for Date objects stored in Firestore is going to change
```

Comment:
1. After seeing this message in the logs, wrapping it to 80 columns seemed like a good idea.
2. This message gets logged about a thousand times during tests. It's super-annoying, do you have any suggestions on how to disable it during a test run?

Reply:
1. Hrm.  I suspect wrapping to 80 columns would be friendly on the other platforms, though I don't remember if Xcode and Android Studio wrap automatically in their consoles or not.  So when you revisit the log messages pending API review feedback, etc., maybe wrap them, but I am not _too_ concerned about it.
2. I think the 11 tests should be changed to expect the new behavior, but I'd make sure we have at least one test that verifies server timestamps return Dates (for ServerTimestampBehavior 'estimate') if you have the enableTimestampsInSnapshots setting turned off.

Reply:
Changed the tests, including one test for server timestamps that previously checked for Timestamps if the setting is true, but now checks for Dates if setting is false.